src/Microsoft.Framework.PackageManager/Commands/WrapCommand.cs

```diff
@@ -47,7 +47,7 @@ public bool ExecuteCommand()
             var extension = Path.GetExtension(InputFilePath);
             if (string.Equals(extension, ".csproj", StringComparison.OrdinalIgnoreCase))
             {
-                if (PlatformHelper.IsMono)
+                if (!RuntimeEnvironmentHelper.IsWindows)
```

Comment:
Curious: What are there rules by which you've chosen `IsMono` -> `IsMono` in some places and `IsMono` -> `!IsWindows` in others?